src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.service;
 
+import static mate.academy.service.UserService.users;
+
+import mate.academy.model.User;
+
 public class AuthenticationService {
```

Comment:
You should use the `UserService` and method `findByEmail` here

Reply:
"Your task is to implement a login method in AuthenticationService and findByEmail method in UserService. More details are described in comments over the methods. "  -  this condition in file "README.md".
 I thought I should implement the method body in class, were method create.